en/sensor/ulanding_radar.md

```diff
@@ -0,0 +1,46 @@
+# uLanding Radar
+
+The *Aerotenna* [uLanding](https://aerotenna.com/sensors/#ulanding) altimeter is compact microwave rangefinder that has been optimised for use on UAVs. With a sensing range of 45m, it is useful for applications including terrain following, precision hovering (e.g. for photography), anti-collision sensing etc. Particular advantages of this product are that it can operate effectively in all weather conditions and over all terrain types (including water).
+
+<img src="../../assets/hardware/sensors/uLanding_lite_1.jpg" alt="Aerotenna uLanding" width="300px" />
+
+> **Note** Due to flash space limitations on some flight controllers the driver may not be built into all firmware. 
```

Comment:
This is in nuttx_px4fmu-v2_default.cmake - does that mean it is in all FMUv2 boards - e.g. Pixhawk1 ?

Reply:
Yes but it's more complicated: some newer boards that support the 2MB flash can use the nuttx_px4fmu-v3_default.cmake target (e.g. pixhawk mini)

Reply:
So basically it would appear in FMUv2 or FMUv3 boards that use the FMUv2 firmware? Otherwise you need to add it yourself?

Reply:
It would appear in FMUv2 or FMUv3 boards that use the FMUv**3** firmware

Yes otherwise you have to add it yourself and most likely remove some other unused driver.

Reply:
That is a little odd, because the driver is in "This is in nuttx_px4fmu-v2_default.cmake", not the FMuv3 config !

At this point I won't specify - below I've explained how they can add/check whether it is present in a configuration.